### PR TITLE
14.0 fix option duplication

### PR DESCRIPTION
**Pull request by mourad-ehm**
ping @sebastienbeau

### CODE REVIEW

File: sale_configurator_option/models/sale.py [hunk 1]
```diff
@@ -44,6 +68,23 @@ class SaleOrderLine(models.Model):
         ondelete="set null",
         compute="_compute_product_option_id",
     )
+    origin_copy_id = fields.Many2one(
```
Comment (sebastienbeau) [inline sale_configurator_option/models/sale.py:71]: We should not add a new field for this.
We should find an other implementation

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

File: sale_configurator_option/models/sale.py (hunk 1 above)
Comment (mourad-ehm) [reply sale_configurator_option/models/sale.py:71]: Where is -the problem ? In odoo there are many technical fields

Comment (mourad-ehm) [reply sale_configurator_option/models/sale.py:71]: ping @sebastienbeau

Comment (mourad-ehm) [reply sale_configurator_option/models/sale.py:71]: I suggest to add this solution until we found an other one (we can improve later). there is a real bug now.

Comment (mourad-ehm) [reply sale_configurator_option/models/sale.py:71]: ping @sebastienbeau

**Comment by mourad-ehm** [comment]
@paradoxxxzero could you review this PR ?

**Comment by sebastienbeau** [comment]
replaced by #60

**Pull request closed without merging**